### PR TITLE
[ADD] event_project_issue: Project issuew in events, and in event ses…

### PR DESCRIPTION
**Pull request by alfredoavanzosc**
…sions.

Nuevo módulo para asociar incidencias de proyecto a eventos, y sesiones de evento.

### CODE REVIEW

File: event_project_issue/models/event.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+class EventEvent(models.Model):
+    _inherit = 'event.event'
+
+    @api.multi
+    @api.depends('issues')
+    def _count_event_issues(self):
+        for event in self:
+            event.count_issues = len(event.issues)
+
+    issues = fields.One2many(
```
Comment (oihane) [inline event_project_issue/models/event.py]: issue_ids

File: event_project_issue/models/event.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+class EventTrack(models.Model):
+    _inherit = 'event.track'
+
+    @api.multi
+    @api.depends('issues')
+    def _count_session_issues(self):
+        for session in self:
+            session.count_issues = len(session.issues)
+
+    issues = fields.One2many(
```
Comment (oihane) [inline event_project_issue/models/event.py]: issue_ids

File: event_project_issue/models/project_issue.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# (c) 2016 Alfredo de la Fuente - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from openerp import models, fields, api
+
+
+class ProjectIssue(models.Model):
+    _inherit = 'project.issue'
+
+    event = fields.Many2one(
```
Comment (oihane) [inline event_project_issue/models/project_issue.py]: event_id

File: event_project_issue/models/project_issue.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        comodel_name='event.event', string="Event")
+    session = fields.Many2one(
```
Comment (oihane) [inline event_project_issue/models/project_issue.py]: session_id

**Comment by oihane** [comment]
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#field
- One2Many and Many2Many fields should always have _ids as suffix (example: sale_order_line_ids)
- Many2One fields should have _id as suffix (example: partner_id, user_id, ...)

**Comment by alfredoavanzosc** [comment]
Realizadas las mejoras solicitadas por @oihane , y subido el módulo.

**Pull request merged**